src/vivarium/interpolation.py

```diff
@@ -28,9 +28,7 @@ class Interpolation:
         continuous_parameters :
             Column names to be used as continuous parameters in Interpolation. If
             bin edges, should be of the form (column name used in call, column name
-            for left bin edge, column name for right bin edge). If order is 0 and
-            any continuous parameter is given without bin edges, continuous bins
```

Comment:
In going through, realized this was still hanging around but is not true - if any continuous parameter is given without bin edges, interpolation will break and yell at you